src/components/Books.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import BookDetail from "./BookDetail";
+import BookAction from "./BookAction";
+import BookProgress from "./BookProgress";
+
+const Books = () => {
+  return (
+    <div>
+      <BookDetail />
+      <BookAction />
+      <BookProgress />
+    </div>
+  );
```

Comment:
Hello @DilsherB 
- [ ] As requested in the project requirements, let make sure below features are implemented:
- Contains individual book state, as well as a button to delete the book (the button wouldn't remove the books *yet*, this is just a placeholder for now)
- Contains a list to renders individual books
- Contains a form to create new books (with inputs for the title, the author of the book and a button to submit)
   the button wouldn't add the book *yet*, this is just a placeholder for now